docs/book/src/clusterctl/developers.md

```diff
@@ -135,14 +147,40 @@ nodes:
         containerPath: /var/run/docker.sock
 EOF
 ```
+- Run `kind create cluster --config kind-cluster-with-extramounts.yaml` to create the management cluster using the above file
+
+- Run `kind load docker-image gcr.io/k8s-staging-cluster-api/capd-manager-amd64:dev` to make the docker provider image available for the kubelet in the management cluster.
+
+- Run `clusterctl init` command provided as output of the local-overrides hack.
 
+{{#/tab }}
+{{#tab v0.8.x}}
+
+- Export the **KIND_EXPERIMENTAL_DOCKER_NETWORK=bridge**
```

Comment:
```suggestion
- Export the **KIND_EXPERIMENTAL_DOCKER_NETWORK=bridge** because so kind and CAPD are going to create containers in the same network (Since **kind v0.8.0**, kind by default is creating the containers in the **kind** network while CAPD is creating them in the **bridge** one)
```

Reply:
done